src/Adapter/MSTestAdapter.PlatformServices/ObjectModel/UnitTestElement.cs

```diff
@@ -246,7 +240,26 @@ private Guid GenerateSerializedDataStrategyTestId(string testFullName)
             hash.Append(CloseBracket);
         }
 
-        return new Guid(hash.GetCurrentHash());
+        byte[] hashBytes = hash.GetCurrentHash();
+
+        // https://www.rfc-editor.org/rfc/rfc9562.html#name-uuid-version-8
+        var guid = new Guid(hashBytes);
```

Comment:
Your comment on the PR says:

In addition, XxHash128 output is big endian, so it's best if we do new Guid(hash, bigEndian: true) instead of new Guid(hash).


```suggestion
        // XxHash128 output is big endian
        var guid = new Guid(hashBytes, bigEndian: true);
```

Reply:
`bigEndian` parameter wasn't available everywhere, so I preferred to use the same code across all TFMs.

Reply:
Corrected for that in code.